src/gs-details-page.c

```diff
@@ -2726,9 +2729,8 @@ gs_details_page_setup (GsPage *page,
 	self->plugin_loader = g_object_ref (plugin_loader);
 	self->builder = g_object_ref (builder);
 	self->cancellable = g_object_ref (cancellable);
-
-	self->plugin_status = GS_PLUGIN_STATUS_FINISHED;
-
+	self->plugin_statuses = g_hash_table_new (g_str_hash,
```

Comment:
Likewise, this needs to be disposed and its values packed and unpacked from pointers using the macros described above.

Reply:
No, I think we need to use `gs_plugin_loader_get_state()`. Why would we make a separate hash table here?

Reply:
This seems to be resolved in the latest version